components/behaviors/time-series.schema.json

```diff
@@ -55,7 +55,9 @@
             "commerce.saveForLaters": "Commerce Save For Laters",
             "delivery.feedback": "Delivery Feedback",
             "message.feedback": "Message Feedback",
-            "message.tracking": "Message Tracking"
+            "message.tracking": "Message Tracking",
+            "pushTracking.applicationOpened": "Push notification launched application",
```

Comment:
@iewilliam  - Will you update the values so they match the formatting of other meta:enum's

Reply:
Updated it just now, will this work?
            "pushTracking.applicationOpened": "Push triggered application launches",
            "pushTracking.customAction": "Push custom action clicks"